core/platform/taskqueue/gae_taskqueue_services.py

```diff
@@ -36,6 +38,13 @@ def defer_to_events_queue(fn, *args, **kwargs):
     # details on the _queue kwarg.
     deferred.defer(fn, *args, _queue=QUEUE_NAME_EVENTS, **kwargs)
 
+def add_feedback_message_email_task(url, **kwargs):
```

Comment:
Oh so you suggest that rather than kwargs, we should have all args explicitly mentioned.
That sounds good to me.